saleor/graphql/checkout/tests/mutations/test_checkout_shipping_method_update.py

```diff
@@ -683,7 +810,7 @@ def test_checkout_shipping_method_update_external_shipping_triggers_webhooks(
     assert mocked_send_webhook_request_async.call_count == 1
 
     # confirm each sync webhook was called without saving event delivery
-    assert mocked_send_webhook_request_sync.call_count == 5
+    assert mocked_send_webhook_request_sync.call_count == 3
```

Comment:
It is expected change. The below logic validates only 3 sync webhook calls. Previously we had duplicate calls